graphblas/io.py

```diff
@@ -183,15 +287,8 @@ def mmwrite(target, matrix, *, comment="", field=None, precision=None, symmetry=
         from scipy.io import mmwrite
     except ImportError:  # pragma: no cover
         raise ImportError("scipy is required to write Matrix Market files") from None
-
-    # Note: it would be preferable to write the data in full format if it is indeed full.
-    #       However, there may be a bug: https://github.com/scipy/scipy/issues/13634
-    #
-    # SS, SuiteSparse-specific: format
-    # fmt = matrix.ss.format
-    # if fmt == 'fullr' or fmt == 'fullc':
-    #    array = matrix.ss.export(format=fmt)['values']
```

Comment:
btw, I uncommented these lines and changed the minimum required version of scipy to 1.7.0, which is the first release with the fix for the issue in the code comment above.